Formula/tfproviderlint.rb

```diff
@@ -18,6 +18,9 @@ class Tfproviderlint < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "76c181e440e93dca6784ff49c7592f2a039ba64785220c7f9eb66bdc73f3273a"
   end
 
+  # Issue ref: https://github.com/bflad/tfproviderlint/issues/255
+  deprecate! date: "2023-02-14", because: "errors with Go 1.18 or later"
```

Comment:
Tried locally and hit:
```
==> /usr/local/Cellar/tfproviderlint/0.28.1_1/bin/tfproviderlint -fix /private/tmp/tfproviderlint-test-20230214-6254-4iqwjf/... 2>&1
tfproviderlint: internal error: package "log" without types was imported from "github.com/russellcardullo/terraform-provider-pingdom/pingdom"
```

Which seems like reported issue. There are a number of open PRs to update `x/tools` but not much progress.

Can re-enable if upstream does merge a fix within the 1-2yr timeframe of removal.